wavefront_cli/lib/agent.py

```diff
@@ -23,24 +23,21 @@ def get_install_agent_cmd():
               " support@wavefront.com.")
         return cmd
 
-    if dist.strip() in ("Oracle Linux Server", "Fedora",
-                        "Red Hat Enterprise Linux Server",
-                        "Red Hat Enterprise Linux Workstation",
-                        "CentOS", "CentOS Linux"
-                        ) or dist.startswith("Amazon Linux"):
+    if dist.strip().startswith(("Oracle Linux Server", "Fedora",
+                                "Amazon Linux", "CentOS",
+                                "Red Hat Enterprise Linux")):
         cmd = "curl -s %s | bash" % (agent_pkg_rpm)
         cmd += " && yum -y -q install telegraf"
-    elif dist == "Ubuntu":
+    elif dist.strip().startswith("Ubuntu"):
         cmd = "curl -s %s | bash" % (agent_pkg_deb)
         cmd += ' && apt-get -y -qq -o Dpkg::Options::="--force-confold"' \
                ' install telegraf'
-    elif dist == "debian":
+    elif dist.strip().startswith("Debian"):
```

Comment:
Here and elsewhere you may also add `lower()` to disambiguate the capitalization, e.g.
```dist.strip().lower().startswith('debian')```

Reply:
distro is returning the **Debian GNU/Linux** as OS name for Debian OS
https://github.com/nir0s/distro/blob/master/tests/test_distro.py#L181

Reply:
@yogeshprasad I see it used to be lower case `"debian"` and if it used to work it means in some cases it may vary